tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -663,36 +672,32 @@ methods that have changed so don't remove anything we're omitting:
 Now when the project is compiled we'll see another property called speed.
 Changing its value will make the animation go faster or slower.
 
-For this example there is no obvious advantage of using a setter and getter.
-It is just more code to write. For a simple example as this there may be a
-good reason for a setter if you want to react on the variable being changed
-but in many cases just binding the variable will be enough.
+For this example there is no obvious advantage of using a setter and getter. It
+is just more code to write. For a simple example as this there may be a good
+reason for a setter if you want to react on the variable being changed but in
+many cases just binding the variable will be enough.
 
-Getters and setters become far more useful in more complex scenarios
-where you need to make additional choices based on the state of your
-object.
+Getters and setters become far more useful in more complex scenarios where you
+need to make additional choices based on the state of your object.
 
 .. note::
 
     For simplicity we've left out the optional parameters in the
     register_property<class, type> method call. These parameters are
-    ``rpc_mode``, ``usage``, ``hint`` and ``hint_string``. These can
-    be used to further configure how properties are displayed and set
-    on the Godot side.
+    ``rpc_mode``, ``usage``, ``hint`` and ``hint_string``. These can be used to
+    further configure how properties are displayed and set on the Godot side.
 
-    Modern C++ compilers are able to infer the class and variable type
-    and allow you to omit the ``<GDExample, float>`` part of our
-    ``register_property`` method. we've had mixed experiences with this
-    however.
+    Modern C++ compilers are able to infer the class and variable type and allow
+    you to omit the ``<GDExample, float>`` part of our ``register_property``
+    method. we've had mixed experiences with this however.
```

Comment:
Shouldn't this sentence start with a capitalized word?

Reply:
Definitely. I haven't proofread that page yet, only rewrapped it. Full proofreading is my next step :)